### DISCUSSION TITLE
Use carets for dependencies versions

### DISCUSSION THREAD

**Discussion by oceandrama** [chakra-ui/chakra-ui#9418] [General] [answered]
### Description

Now `@chakra-ui/react` has strict dependencies:
```json
"@emotion/is-prop-valid": "1.3.1",
"@emotion/serialize": "1.3.2",
"@emotion/use-insertion-effect-with-fallbacks": "1.1.0",
"@emotion/utils": "1.4.1"
```
It leads to duplication of dependencies from `@emotion/react`, which has higher versions
```json
"@emotion/babel-plugin": "^11.13.5",
"@emotion/cache": "^11.14.0",
"@emotion/serialize": "^1.3.3",
"@emotion/use-insertion-effect-with-fallbacks": "^1.2.0",
"@emotion/utils": "^1.4.2",
"@emotion/weak-memoize": "^0.4.0"
```

### Problem Statement/Justification

Dependencies duplication negatively impact packages loading time, node_modules size and most importantly bundle size

### Proposed Solution or API

Using carets (^) for dependencies versions could fix this problem

**Comment by segunadebayo** [comment, marked as answer]
Using strict dependencies was an intentional decision to ensure the contributors can reproduce issues and avoid version mismatch when downstream packages are released.

I recommend using npm or pnpm resolutions in your codebase instead. On our side, we'll do our best to keep the emotion dependencies up to date.
